plasmapy/physics/transport/collisions.py

```diff
@@ -633,27 +633,34 @@ def collision_frequency(T,
     .. [2] http://homepages.cae.wisc.edu/~callen/chap2.pdf
     """
     # boiler plate checks
-    T, masses, charges, reduced_mass, V = _boilerPlate(T=T,
-                                                       particles=particles,
-                                                       V=V)
-    if particles[0] == 'e' and particles[1] == 'e':
+    T, masses, charges, reduced_mass, V_red = _boilerPlate(T=T,
+                                                           particles=particles,
+                                                           V=V)
+    if particles[0] in ('e','e-') and particles[1] in ('e','e-'):
+        # if a velocity was passed, we use that instead of the reduced
+        # thermal velocity
+        if np.isnan(V):
+            V = V_red
```

Comment:
I got slightly confused by the abbreviated variable name `V_red`.  I think it would more readable if it's changed to `V_reduced`.

Reply:
It'll make line 636 too long... but I have a cheeky compromise...